ch32v003fun/ch32v003fun.h

```diff
@@ -13794,6 +13794,10 @@ void SystemInit(void);
 // Put an output debug UART on Pin D5.
 // You can write to this with printf(...) or puts(...)
 
+#include <setjmp.h>
```

Comment:
Why do you #include this here?  Don't you only need the forward definitions?  (I'd rather not blindly #include extra headers.

Reply:
Why repeat the typedef logic?, i could inlcude <machine/setjmp.h> which just contains the definition for jmp_buf.